seagull/lifeforms/gliders.py

```diff
@@ -16,3 +16,27 @@ def __init__(self):
     @property
     def layout(self) -> np.ndarray:
         return np.array([[1, 0, 0], [0, 1, 1], [1, 1, 0]])
+		
+
+class LightweightSpaceship(Lifeform):
+	def __init__(self):
+		super(LightweightSpaceship, self).__init__()
+
+	@property
+	def layout(self) -> np.ndarray:
```

Comment:
Please run this file onto `black` so that it's formatted correctly :) 
Also, I think you need to include these classes in the `__init__.py` file so that it's detected by the test runner

Thank you so much!

Reply:
Hi! Thanks for helping. This is my first ever contribution to somebody else's project :D 
I have made the changes, committed and pushed to my branch. As far as I understand, the changes appear automatically and I don't need to make a new pull request? 
Thanks!

Reply:
Yup you are right! I appreciate the contribution, I hope you're having fun! :+1: I'll just make a few more updates on my end then we can merge :+1:  All changes now pass, I'll merge this tomorrow!